mobileChatExamples/androidChatExample/README.md

```diff
@@ -89,6 +89,44 @@ The WebsocketManager handles the WebSocket connection lifecycle and receives cha
 - **websocketDidConnect**: Called when the WebSocket connects, and may subscribe to topics if necessary.
 - **websocketDidReceiveMessage**: Parses and handles incoming messages, delegating them back to ChatManager for UI updates.
 
+
+### Chat Rehydration
+
+Chat rehydration is a feature that allows users to continue their previous chat sessions. This process involves several checks and actions:
+
+- **Check for Participant Token:**
+  On initiating chat, the module first checks if a `participantToken` exists.
+  - If it exists, the module proceeds to fetch the chat transcript, allowing the user to continue from where they left off.
+  - If it does not exist, the module then checks for a `contactId`.
+
+- **Use of Contact ID:**
+  If a `contactId` exists, the module prompts the user to either restore the previous session or start a new chat.
+  - If the user chooses to restore, the module starts a new chat session with the existing `contactId`, creates a new participant connection, and then fetches the transcript.
```

Comment:
Can we mention that there will be a new initialContactId when chat is rehydrated?  The existing contactId will only be used as the SourceContactId. Customer needs to use the new initialContactId for the CreateParticipantConnection call or other APIs if they want to operate on the new contact. 
Missed that in the PR for iOS example, can you update there as well - https://github.com/amazon-connect/amazon-connect-chat-ui-examples/blob/master/mobileChatExamples/iOSChatExample/README.md?

Reply:
Sure, good point!